test/cli/package-prefix-enforcement/test.out

```diff
@@ -27,10 +27,6 @@ nested/nested.rb:38: File belongs to package `Root::Nested` but defines a consta
      3 |class Root < PackageSpec
         ^^^^^^^^^^^^^^^^^^^^^^^^
 
-nested/nested.rb:41: Class or method behavior may not be defined outside of the enclosing package namespace `Root::Nested` https://srb.help/3713
-    41 |  def self.method
-          ^^^^^^^^^^^^^^^
```

Comment:
We already issued an error for this several lines above, so not having this error is an improvement.  A previous commit fixed `test/testdata/packager` tests (which also had duplicate errors prior to this PR), but I forgot to run the CLI tests locally prior to doing that commit.